app/components/search/Sidebar/Sidebar.module.scss

```diff
@@ -0,0 +1,151 @@
+@import "~styles/utils/_helpers.scss";
+
+@keyframes slideRight {
+  from {
+    transform: translateX(-100%);
+  }
+
+  to {
+    transform: translateX(0);
+  }
+}
+
+@keyframes slideUp {
+  from {
+    transform: translateY(100%);
+  }
+
+  to {
+    transform: translateY(70px);
+  }
+}
+
+.sidebar {
+  flex-shrink: 0;
+  position: sticky;
+  height: calc(100vh - 70px);
+  width: 290px;
+  top: $header-height;
+  border-right: 1px solid $color-grey3;
+  padding: 20px 25px 50px 30px;
+  overflow-y: auto;
+
+  @media screen and (max-width: $break-tablet-l) {
+    height: auto;
+    width: 60%;
+    padding: 0 30px;
+    z-index: 2;
```

Comment:
I've removed the CSS rule in the .showFilters state below, but kept it here since the filters bar is sticky and scrolls down with the page so it needs to overlay the search results like so:
![image](https://user-images.githubusercontent.com/3012520/162078433-992267cd-b7bd-43a9-bbb1-5e9b303c7c4f.png)
(I will double check with the designers if they wanted it to scroll down but that was my assumption)